CardanoSharp.Wallet/Models/WalletPath.cs

```diff
@@ -0,0 +1,107 @@
+﻿using CardanoSharp.Wallet.Enums;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace CardanoSharp.Wallet.Models
+{
+    /// <summary>
+    /// Path used in Hierarchical Deterministic (HD) Wallets for Cardano 
+    /// See <see href="https://cips.cardano.org/cips/cip1852/">CIP1852</see> for context
+    /// </summary>
+    public class WalletPath
+    {
+        private readonly DerivationType[] _derivations = new DerivationType[]
+        {
+            DerivationType.SOFT, // m
+            DerivationType.HARD, // purpose
+            DerivationType.HARD, // coin
+            DerivationType.HARD, // account
+            DerivationType.SOFT, // role
+            DerivationType.SOFT  // index
+        };
```

Comment:
Can we use this for anything?

Reply:
I definitely like the idea of this. it helps enforce the rules of each derivation. 🤔